test/Microsoft.ML.Tests/TrainerEstimators/MatrixFactorizationTests.cs

```diff
@@ -62,13 +66,16 @@ public void MatrixFactorizationSimpleTrainAndPredict()
             var data = reader.Read(new MultiFileSource(GetDataPath(TestDatasets.trivialMatrixFactorization.trainFilename)));
 
             // Create a pipeline with a single operator.
-            var pipeline = mlContext.Recommendation().Trainers.MatrixFactorization(userColumnName, itemColumnName, labelColumnName,
-                advancedSettings: s =>
-                {
-                    s.NumIterations = 3;
-                    s.NumThreads = 1; // To eliminate randomness, # of threads must be 1.
-                    s.K = 7;
-                });
+            var options = new MatrixFactorizationTrainer.Options {
+                MatrixColumnIndexColumnName = userColumnName,
+                MatrixRowIndexColumnName = itemColumnName,
+                LabelColumnName = labelColumnName,
+                NumIterations = 3,
+                NumThreads = 1,
```

Comment:
Please copy my old comment. Thanks.

Reply:
added back

---
In reply to: [250022503](https://github.com/dotnet/machinelearning/pull/2204#discussion_r250022503) [](ancestors = 250022503)